components/sup/src/manager/service/mod.rs

```diff
@@ -27,7 +27,7 @@ use std::io::prelude::*;
 use std::path::{Path, PathBuf};
 use std::result;
 use std::sync::{Arc, RwLock};
-use std::time::Instant;
+use std::time::{Instant, Duration};
```

Comment:
Either run `cargo fmt` (or set up rustfmt to automatically run in your editor) or fix this one by hand to make travis pass
```suggestion
use std::time::{Duration, Instant};
```